miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -102,6 +105,8 @@ abstract class MiniAppMessageBridge {
             ActionType.SHARE_INFO.action -> onShareContent(callbackObj.id, jsonStr)
             ActionType.LOAD_AD.action -> onLoadAd(callbackObj.id, jsonStr)
             ActionType.SHOW_AD.action -> onShowAd(callbackObj.id, jsonStr)
+            ActionType.GET_USER_NAME.action -> userInfoHandler.onGetUserName(callbackObj.id)
```

Comment:
Just one thing here and below: it's possible the Host App didn't call `setUserInfoHandler`, so we need to handle that case. Maybe either a default implementation of `UserInfoHandler` that just returns an error or just directly return an error here?

Reply:
Ah, I see you're already throwing errors from the default methods, so I guess you could just remove `lateinit` from `userInfoHandler` variable and directly instantiate the default `UserInfoHandler()`.